app/assets/v2/css/rain.css

```diff
@@ -1,763 +1,155 @@
 .rain {
+  position: fixed;
+  top: -50%;
   background: white;
   height: 100px;
-  position: absolute;
   width: 6px;
-  transform: rotate(40deg);
-  border-radius: 5px;
+  border-radius: 15px;
+  z-index: 0;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  cursor: default;
+  will-change: transform, top, left;
+  -webkit-animation-name: rains-fall, rains-shake;
```

Comment:
Just for being safe, do we add `-moz` and `-o` prefix as well?

Reply:
Well is using caniuse data to determine if need to prefix it, we can add but since is a visual stuff and for evergreen browsers I don't know if make sense.

Reply:
90.62% of web is keyframe unprefixed https://caniuse.com/#search=keyframe But tell me if you think is better to get safe I add it